schematic/models/metadata.py

```diff
@@ -74,6 +74,21 @@ def __init__(self, inputMModelLocation: str, inputMModelLocationType: str,) -> N
                 f"The type '{inputMModelLocationType}' is currently not supported."
             )
 
+    def parse_project_scope(self,project_scope):
```

Comment:
Start a cmd_utils.py and add the parser function there as parse_synid_list() (or a different better name? :)